src/NumericalAlgorithms/Spectral/SwshDerivatives.hpp

```diff
@@ -0,0 +1,309 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <complex>
+#include <cstddef>
+
+#include "DataStructures/ComplexDataVector.hpp"  // IWYU pragma: keep
+#include "DataStructures/ComplexDiagonalModalOperator.hpp"
+#include "DataStructures/ComplexModalVector.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/SpinWeighted.hpp"
+#include "DataStructures/TempBuffer.hpp"  // IWYU pragma: keep
+#include "DataStructures/Variables.hpp"
+#include "NumericalAlgorithms/Spectral/ComplexDataView.hpp"
+#include "NumericalAlgorithms/Spectral/SwshCoefficients.hpp"
+#include "NumericalAlgorithms/Spectral/SwshCollocation.hpp"
+#include "NumericalAlgorithms/Spectral/SwshTags.hpp"
+#include "NumericalAlgorithms/Spectral/SwshTransform.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+// IWYU pragma: no_forward_declare ComplexDataVector
+// IWYU pragma: no_forward_declare SpinWeighted
+// IWYU pragma: no_forward_declare Variables
+
+namespace Spectral {
+namespace Swsh {
+namespace detail {
+
+const size_t swsh_derivative_maximum_l_max = 200;
```

Comment:
`constexpr`

Would it be worth factoring this and the other control parameter like this (I don't remember where it's at) into a separate file if they're both in `NA/Spectral/`? Something like `SwshSettings.hpp` :man_shrugging: